meaningful/stats.py

```diff
@@ -1,3 +1,9 @@
 def mean(l):
     n = len(l)
     return 1.0 / n * sum(x for x in l)
+
+def variance(l):
+    n = len(l)
+    mu = mean(l)
+    print('Hello you!')
```

Comment:
please remove print statements

Reply:
yes I did! thanks for your review! :kiss: